### PR TITLE
Improve readability of the `index describe`

### PR DESCRIPTION
**Pull request by boraarslan**
## CSS is easier

Disclaimer: I _tried_ to push a good commit history; however, git is my greatest enemy. I failed miserably.

The new version looks like this:
image

Changes:
- Well, I think it satisfies every criterion on the #3999 
- Updated `tabled` version
- Uses `chrono` for parsing the timestamp

I know the question that everyone who sees the screenshot will ask. It's blatant;

**Q: WOW! That font looks so COOL! You must be a really cool person! A retro monospace font with some modern vibes to it?!?!?**
A: I know, I know. I really am cool. Thanks for asking. The font is [Berkeley Mono](https://berkeleygraphics.com/typefaces/berkeley-mono/). No I'm not sponsored

_Hopefully closes #3999_

### CODE REVIEW

**Comment by fmassot** [comment]
you're soooooo cool @boraarslan!

File: quickwit/quickwit-cli/src/index.rs [hunk 1]
```diff
@@ -667,90 +705,137 @@ impl IndexStats {
...
-                std_val: size_stats.std_val / 1_000_000.0,
-            };
-            let size_stats_table = create_table(size_stats_in_mb, "Size in MB stats (published)");
-            index_stats_table.with(Concat::vertical(size_stats_table))
-        } else {
-            index_stats_table
-        };
+        if let Some(size_stats) = &self.num_bytes_descriptive {
+            let size_stats_in_mb = size_stats / 1_000_000.0;
+            let size_stats_table = size_stats_in_mb.into_table("Size in MB stats (published)");
```
Comment (guilload) [inline quickwit/quickwit-cli/src/index.rs:719]:
```suggestion
            let size_stats_table = size_stats_in_mb.into_table("Published splits size stats (MB)");
```

File: quickwit/quickwit-cli/src/index.rs [hunk 2]
```diff
@@ -544,48 +549,81 @@ pub struct IndexStats {
 }
 
 impl Tabled for IndexStats {
-    const LENGTH: usize = 7;
+    const LENGTH: usize = 9;
 
-    fn fields(&self) -> Vec<String> {
-        vec![
-            self.index_id.clone(),
+    fn fields(&self) -> Vec<Cow<'_, str>> {
```
[the remark this answers is not in the document]
Comment (boraarslan) [reply quickwit/quickwit-cli/src/index.rs:554]: It's the new trait definition: https://docs.rs/tabled/latest/tabled/trait.Tabled.html; I would've preferred the `String` too.

File: quickwit/quickwit-cli/src/index.rs [hunk 3]
```diff
@@ -544,48 +549,81 @@ pub struct IndexStats {
...
-            "Index URI: ".to_string(),
-            "Number of published documents: ".to_string(),
-            "Size of published documents (uncompressed): ".to_string(),
-            "Number of published splits: ".to_string(),
-            "Size of published splits: ".to_string(),
-            "Timestamp field: ".to_string(),
-            "Timestamp range: ".to_string(),
+    fn headers() -> Vec<Cow<'static, str>> {
+        [
+            "Index ID: ",
```
[the remark this answers is not in the document]
Comment (boraarslan) [reply quickwit/quickwit-cli/src/index.rs:579]: on all of them or just from "Index ID"?

[the remark this answers is not in the document]
Comment (guilload) [reply quickwit/quickwit-cli/src/index.rs:579]: All.

File: quickwit/quickwit-cli/src/index.rs (hunk 2 above)
[the remark this answers is not in the document]
Comment (guilload) [reply quickwit/quickwit-cli/src/index.rs:554]: Bummer :)

File: quickwit/quickwit-cli/src/index.rs [hunk 4]
```diff
@@ -544,48 +549,81 @@ pub struct IndexStats {
...
 
-fn display_timestamp_range(range: &Option<(i64, i64)>) -> String {
-    match range {
-        Some((timestamp_min, timestamp_max)) => {
-            format!("{timestamp_min} -> {timestamp_max}")
+fn display_timestamp(timestamp: &Option<i64>) -> String {
+    match timestamp {
+        Some(timestamp) => {
+            let datetime = chrono::NaiveDateTime::from_timestamp_millis(*timestamp * 1000)
+                .map_or("Invalid timestamp!".to_string(), |datetime| {
```
[the remark this answers is not in the document]
Comment (boraarslan) [reply quickwit/quickwit-cli/src/index.rs]: lol, fixed

**Pull request merged**